ModularTegustation/tegu_items/representative/research/lcorp.dm

```diff
@@ -231,3 +231,35 @@ GLOBAL_LIST_EMPTY(lcorp_upgrades)
 	cost = AVERAGE_RESEARCH_PRICE
 	corp = L_CORP_REP
 	required_research = /datum/data/lc13research/salesspeed/lvl7
+
+//LOB points
+/datum/data/lc13research/lob/lvl1
+	research_name = "LOB points grant"
+	research_desc = "HQ is allowing you more LOB points budget in exchange for PE."
+	cost = HIGH_RESEARCH_PRICE
```

Comment:
The Price is way too cheap for 5 points. 5 points is absolutely massive. If you make it 1 or 2 then I'd be way more ok with it.

Reply:
Modified to 2 and 2 as discussed in PR-Discussions.